.travis.yml

```diff
@@ -181,18 +187,38 @@ jobs:
       - bin/release_docker $TRAVIS_TAG
       - if [[ $TRAVIS_TAG != *-rc* ]] && [[ $TRAVIS_TAG != *-dev ]]; then bin/release_docker latest; fi
       name: "Pushing release to hub.docker.com"
+
     - script:
       - bin/s3 sync s3://travis-$TRAVIS_BUILD_NUMBER/build-artifacts build/package
       - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
       - gpg --import bin/package/android/key.gpg
-      - gpg --list-keys
       - bin/release_android "$TRAVIS_TAG"
       name: "Publish Android SDK to the Sonatype and Maven Central"
+
     - script:
       - bin/travis_scripts/trigger_desktop_app_build
       name: "Trigger Desktop APP build"
       if: tag = env(BUILD_DEV_RELEASE)
 
+    - script:
+      - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
```

Comment:
Maybe it makes sense to take these steps and put them in a script in bin folder with a short explanation of variables needed? If we're moving to another CI would make it a bit less frustrating to move

Reply:
It's totally Travis file encryption process: https://docs.travis-ci.com/user/encrypting-files/
Migration to the new CI will require it to be changed anyway. So keeping it in Travis config.